trikot-kword/kword/src/jsMain/kotlin/com.mirego.trikot.kword.js/JsKWord.kt

```diff
@@ -1,39 +1,22 @@
 package com.mirego.trikot.kword.js
 
 import com.mirego.trikot.kword.I18N
-import com.mirego.trikot.kword.KWord
-import kotlinext.js.require
+import com.mirego.trikot.kword.KwordLoader
 
 object JsKWord {
+    private val DEFAULT_PATHS = listOf(
+        "translation"
```

Comment:
You don't need the sub-path `translations` for the Web?

Reply:
You are referring to this? `require("./translations/$path")`.  Would it not work if you changed it to `require(".$path")` and use `/translations/translation` as the given default?

Reply:
exactly, the js compiler extracts the `./translations` part of the string literal and passes it to webpack to include the resources under that folder.